templates/_layouts/docs.html

```diff
@@ -47,7 +47,7 @@
 
             <ul class="p-side-navigation__list">
               <li class="p-side-navigation__item--title"><span class="p-side-navigation__text">Base elements</span></li>
-              {{ side_nav_item("/docs/base/code", "Code") }}
+              {{ side_nav_item("/docs/base/code", "Code", "new") }}
```

Comment:
I tend to put "updated" label in such cases.

While we add a new component, the page itself is not new.

So, I usually mark page as "updated" and in the page content I put "new" label under the section that was added.

Not sure if that's a best approach (because it doesn't it doesn't make it clear in navigation which sections have new stuff and which are just updated), but also doesn't suggest that "Code" is new thing, when it's not.